spec/models/event_spec.rb

```diff
@@ -8,7 +8,6 @@
     expect(event.errors[:description]).to_not be_empty
     expect(event.errors[:website]).to_not be_empty
     expect(event.errors[:start_at]).to_not be_empty
-    expect(event.errors[:end_at]).to_not be_empty
     expect(event.errors[:event_type]).to_not be_empty
```

Comment:
I'm all for having thorough coverage, but testing rails validation seems a bit excessive, especially since it's essentially testing code we don't own. Thoughts?